libpod/oci_conmon_linux.go

```diff
@@ -1160,6 +1160,14 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 	}
 
 	if ctr.config.NetMode.IsSlirp4netns() {
+		havePortMapping := len(ctr.Config().PortMappings) > 0
+		if havePortMapping {
+			ctr.rootlessPortSyncR, ctr.rootlessPortSyncW, err = os.Pipe()
+			if err != nil {
+				return errors.Wrapf(err, "failed to create rootless port sync pipe")
+			}
+		}
+
```

Comment:
should you do the `defer errorhandling` for rootlessPortSyncR/W here as you do below for rootlessSlipSyncR/W around line 1177?

Reply:
no, we cannot close `rootlessPortSyncR` here, it must be left open as it used later